cmake/modules/FindCordova.cmake

```diff
@@ -0,0 +1,8 @@
+find_program(UPM_CORDOVA_BINDING NAMES upm-cordova-binding
```

Comment:
Cordova is an actual npm package (it was the foundation of the Intel XDK), I think it is more appropriate to name this `FindUpmCordovaBindings.cmake`

Reply:
Done.